include/inexor/vulkan-renderer/input/keyboard_mouse_data.hpp

```diff
@@ -75,12 +81,16 @@ class KeyboardMouseInputData {
     /// @param pos_y the current y-coordinate of the cursor
     void set_cursor_pos(double pos_x, double pos_y);
 
-    [[nodiscard]] std::array<std::int64_t, 2> get_cursor_pos() const;
+    [[nodiscard]] glm::ivec2 get_cursor_pos() const;
 
     /// @brief Calculate the change in x- and y-position of the cursor.
     /// @return a std::array of size 2 which contains the change in x-position in index 0 and the change in y-position
     /// in index 1
-    [[nodiscard]] std::array<double, 2> calculate_cursor_position_delta();
+    [[nodiscard]] glm::dvec2 calculate_cursor_position_delta();
```

Comment:
Do we use glm types now everywhere or do we abstract them away to keep independent?
@IAmNotHanni @yeetari

Reply:
I think using glm here is ok.